nondex-maven-plugin/src/it/inexistent-variable-argline-it/verify.groovy

```diff
@@ -0,0 +1,3 @@
+File nondexDirectory = new File( basedir, ".nondex" );
+
+assert ! nondexDirectory.exists();
```

Comment:
Does this verify.groovy check for nonexistent variable in `argLine`? I don't see how